src/components/Person/Person.jsx

```diff
@@ -1 +1,55 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const getmaried = [];
+
+  if (person.isMarried && person.sex === 'f') {
+    getmaried.push('husband');
+  }
+
+  if (person.isMarried && person.sex === 'm') {
+    getmaried.push('wife');
+  }
+
+  if (!person.isMarried && person.age) {
+    getmaried.push('I am not married');
```

Comment:
create a variable for personStatus using `ternary` operator
```suggestion
  const personStatus = sex === 'm' ? 'wife' : 'husband';
```